packages/api-client/src/components/CodeInput/CodeInput.vue

```diff
@@ -257,7 +257,7 @@ export default {
       :id="uid"
       v-bind="$attrs"
       ref="codeMirrorRef"
-      class="font-code peer relative w-full overflow-hidden whitespace-nowrap text-xs leading-[1.44] -outline-offset-1 has-[:focus-visible]:rounded-[4px] has-[:focus-visible]:outline"
+      class="group-[.alert]:outline-orange font-code peer relative w-full overflow-hidden whitespace-nowrap text-xs leading-[1.44] -outline-offset-1 has-[:focus-visible]:rounded-[4px] has-[:focus-visible]:outline"
```

Comment:
Any reason to set the group like this vs a custom tailwind group?

https://tailwindcss.com/docs/hover-focus-and-other-states#differentiating-nested-groups

Reply:
as i relied on an existing `group` utility on the row i was just afraid it would break things to change it 😂 but will do!